src/atlantis/manager/rpc/deploy.go

```diff
@@ -339,8 +342,12 @@ func (e *TeardownExecutor) Execute(t *Task) error {
 			}
 			last, _ := instance.Delete()
 			if last {
+				t.LogStatus("%s is the last one of its kind  [app: %s SHA: %s Env: %s]",
+					tornContainerID, instance.App, instance.Sha, instance.Env)
+   
 				DeleteAppShaFromEnv(instance.App, instance.Sha, instance.Env)
 			}
+			t.LogStatus("successfully teardown %s", tornContainerID)
```

Comment:
Also need to clear tornContainers for every loop... However we need to return a list of torn containers, so maybe we need a new variable here?